src/server/email/sendEmail.js

```diff
@@ -62,14 +108,8 @@ export default async function sendEmailPromise(to, template, props) {
     to,
     message: message.toString('ascii')
   };
-  if (!getMailgunApiConfig().apiKey) {
-    console.warn(`mailgun: no API key, so not sending the following:
-    From: ${from}
-    To: ${to}
-    Subject: ${subject}
-    Body: ${body}
-    `);
-    return true;
-  }
-  return maybeSendMail(mimeData);
+  return maybeMailgun(
+    (mg) => mg.messages().sendMime(mimeData).then(() => true),
```

Comment:
can we remove this `then` statement?

Reply:
I can if you want. I was trying to preserve the existing function's type by preserving the boolean return value.